generators/client/templates/angular/src/main/webapp/app/_vendor.ts

```diff
@@ -6,6 +6,7 @@
 // import 'angular-loading-bar';
 // import 'ng-file-upload';
 // import 'ng-infinite-scroll';
+import 'bootstrap/dist/js/bootstrap.min.js';
```

Comment:
is bootstrap using js to show hide stuff instead of css? I would really like to avoid bootstrap js as we already have ng-bootstrap doing the same stuff in angular way. can we find a way to get these working using ng bootstrap directives